dining/api_wrapper.py

```diff
@@ -0,0 +1,98 @@
+from requests import get
+
+
+class APIError(ValueError):
+    pass
+
+
+def headers():
+    """
+    Returns headers necessary for Penn Dining API access
+    TODO: add the username + password to environment
+    """
+
+    return {
+        "Authorization-Bearer": "adsfdsa",
+        "Authorization-Token": "adsfas",
+    }
+
+
+def dining_request(url):
+    """
+    Makes GET request to Penn Dining API and returns the response
+    """
+
+    response = get(url, params=None, headers=headers(), timeout=30)
+
+    if response.status_code != 200:
+        raise APIError("Request to {} returned {}".format(response.url, response.status_code))
+
+    response = response.json()
+
+    error_text = response["service_meta"]["error_text"]
+    if error_text:
+        raise APIError(error_text)
+
+    return response
+
+
+def normalize_weekly(data):
+    """
+    Normalization for dining menu data
+    """
+
+    if "tblMenu" not in data["result_data"]["Document"]:
+        data["result_data"]["Document"]["tblMenu"] = []
+    if isinstance(data["result_data"]["Document"]["tblMenu"], dict):
+        data["result_data"]["Document"]["tblMenu"] = [data["result_data"]["Document"]["tblMenu"]]
+    for day in data["result_data"]["Document"]["tblMenu"]:
+        if "tblDayPart" not in day:
+            continue
+        if isinstance(day["tblDayPart"], dict):
+            day["tblDayPart"] = [day["tblDayPart"]]
+        for meal in day["tblDayPart"]:
+            if isinstance(meal["tblStation"], dict):
+                meal["tblStation"] = [meal["tblStation"]]
+            for station in meal["tblStation"]:
+                if isinstance(station["tblItem"], dict):
+                    station["tblItem"] = [station["tblItem"]]
+    return data
+
+
+def get_meals(v2_response, building_id):
```

Comment:
Is building_id the same as a Venue's ID? If so we could switch that to take in a venue directly

Reply:
oh woops i'll rename to venue_id for clarity. but i think it should be okay to just pass in the venue_id just because i'm taking the venue_id directly from the URL for that particular route.

also, speaking of: i have the dining api token / password on my computer, but idk how to put it on the api_wrapper file without just writing it flat out for everyone to see. right now i just put some filler text under where i put the request headers, but do you know how to pre-set them to the token / password without making it public on github?